agent/runner/actions/mysql_explain_action.go

```diff
@@ -83,6 +85,9 @@ func (a *mysqlExplainAction) Run(ctx context.Context) ([]byte, error) {
 		return nil, errors.New("Query to EXPLAIN is empty")
 	}
 
+	logrus.Infof("Schema used in explain: %s", a.params.Schema)
```

Comment:
I wonder if those two info log messages should not be invoked with `.Debugf`? Won't these make the logs too verbose?